packages/pyright-internal/src/tests/samples/unreachable2.py

```diff
@@ -0,0 +1,9 @@
+from typing import TYPE_CHECKING
+
+if not TYPE_CHECKING:
+    ...
+
+if TYPE_CHECKING:
+    ...
+else:
+    ...
```

Comment:
might be nice to have an additional 
```py
if TYPE_CHECKING:
    ...
else:
    ...
```

Reply:
the `if not TYPE_CHECKING` above reproduces the same scenario